### PR TITLE
feat: add checkpoint sync capabilities to nimbus

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: src/cl/cl_launcher.star [hunk 1]
```diff
@@ -146,6 +146,26 @@ def launch(
...
+            if checkpoint_sync_url == "":
+                if (
+                    network_params.network in constants.PUBLIC_NETWORKS
+                    or network_params.network == constants.NETWORK_NAME.ephemery
+                ):
+                    plan.print(
+                        "we got here because network is public, {0}".format(
+                            network_params.network
+                        )
+                    )
```
Comment (pk910) [inline src/cl/cl_launcher.star:160]: looks like some leftover from debugging? ;)

File: src/cl/nimbus/nimbus_launcher.star [hunk 2]
```diff
@@ -22,7 +22,7 @@ BEACON_MIN_CPU = 50
 BEACON_MIN_MEMORY = 256
 
 DEFAULT_BEACON_IMAGE_ENTRYPOINT = ["nimbus_beacon_node"]
-
+BEACON_NODE_ENTRYPOINT = "/home/user/nimbus-eth2/build/nimbus_beacon_node"
```
Comment (pk910) [inline src/cl/nimbus/nimbus_launcher.star:25]: It looks very wrong hardcoding this path here as its likely gonna change when nimbus touches their build system.
But yea,  don't really see a better solution :/
Do our ethpandaops nimbus images use the same entrypoint?

File: src/cl/cl_launcher.star (hunk 1 above)
Comment (barnabasbusa) [reply src/cl/cl_launcher.star]: yeah :D

File: src/cl/nimbus/nimbus_launcher.star (hunk 2 above)
Comment (barnabasbusa) [reply src/cl/nimbus/nimbus_launcher.star:25]: yes, works with our images too. 

We can always change it if they decide to change their entrypoint. For now I think this is our "only" reasonable solution.

**Review by pk910 — APPROVED** [review]

**Pull request merged**